rust/src/measurements/gumbel_max/test.rs

```diff
@@ -12,3 +12,29 @@ fn test_exponential() -> Fallible<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_max_vs_min() -> Fallible<()> {
+    assert_eq!(
+        select_score(vec![1, 2, 3].into_iter(), Optimize::Max, RBig::ZERO)?,
+        2
+    );
+    assert_eq!(
+        select_score(vec![1, 2, 3].into_iter(), Optimize::Min, RBig::ZERO)?,
+        0
+    );
+
+    assert_eq!(
+        select_score(vec![1, 1, 100_000].into_iter(), Optimize::Max, RBig::ONE)?,
+        2
+    );
+    assert_eq!(
+        select_score(
+            vec![1, 100_000, 100_000].into_iter(),
+            Optimize::Min,
+            RBig::ONE
+        )?,
+        0
+    );
```

Comment:
I think the idea here is that with a non-zero scale you need large numbers to be fairly confident that the expected number is chosen, and we wanted to test that, too?